### PR TITLE
DCF RELEASE 1592942757

### PR DESCRIPTION
**Pull request by themarcelor**
copying files from nci-crdc-staging.datacommons.io to nci-crdc.datacommons.io

### CODE REVIEW

File: nci-crdc.datacommons.io/manifest.json [hunk 1]
```diff
@@ -32,19 +37,106 @@
...
     "enabled": "yes"
   },
-  "jupyterhub": {
-    "enabled": "no"
+  "ssjdispatcher": {
+    "job_images": {
+      "indexing": "quay.io/cdis/indexs3client:2020.06"
+    }
   },
+  "sower": [
```
Comment (themarcelor) [inline nci-crdc.datacommons.io/manifest.json:45]: @jiaqi216 , do we want to roll this out in this cycle?

File: nci-crdc.datacommons.io/manifests/fence/fence-config-public.yaml [hunk 2]
```diff
@@ -1,91 +1,16 @@
...
-    cred: 'fence_bot'
-  gdc-ohsu-cnl-phs001799-2-open:
-    cred: 'fence_bot'
-  gdc-organoid-pancreatic-phs001611-2-open:
-    cred: 'fence_bot'
-  gdc-varepop-apollo-phs001374-2-open:
+  tcga-protected-dcf-databucket-gen3:
     cred: 'fence_bot'
-  gdc-ccle-2-open:
+  tcga-public-dcf-databucket-gen3:
```
Comment (themarcelor) [inline nci-crdc.datacommons.io/manifests/fence/fence-config-public.yaml:6]: @emalinowski , can you help me tidy this up? 🙏

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- portal
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- portal
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- portal
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- portal
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- portal
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Pull request merged**